cli_docs.md

```diff
@@ -0,0 +1,340 @@
+┃ ID                      ┃ Task Name                                 ┃ Feedback                                                                                         ┃ Completion ID            ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ 4f5fefd1-06a4-415e-b56f │ Demo TLDR Dataset Summary grading w/ note │ {"note": "sounds like the girl is really into OP and OP is wary about committing too easily      │ 0b5bc07c-a976-49cc-bd160 │
+│                         │                                           │ -which is absolutely not the right interpretation.", "overall": 2, "accuracy": 3, "coverage": 2, │                          │
+│                         │                                           │ "coherence": 7}                                                                                  │                          │
+│ 5525bf49-a870-465a-b6b7 │ Demo TLDR Dataset Summary grading w/ note │ {"note": "The summary conveys the main idea of the post.", "overall": 7, "accuracy": 7,          │ 431ee175-3a37-436a-bd627 │
+│                         │                                           │ "coverage": 7, "coherence": 7}                                                                   │                          │
+│ 9874ca7b-1c01-4785-8331 │ Demo TLDR Dataset Summary grading w/ note │ {"note": "•explicit purpose statement will make summary perfect. ", "overall": 6, "accuracy": 7, │ 447c3b69-6aea-4f4d-95846 │
+│                         │                                           │ "coverage": 6, "coherence": 7}                                                                   │                          │
+│ ad7dd317-3a19-4633-bf80 │ Demo TLDR Dataset Summary grading w/ note │ {"note": "Missing details of why.", "overall": 4, "accuracy": 7, "coverage": 4, "coherence": 7}  │ 1d2396bf-df44-4eaf-a9f03 │
+│ 4b5edcb1-ca79-42a8-a0f5 │ Demo TLDR Dataset Summary grading w/ note │ {"note": "Should mention they're male.", "overall": 6, "accuracy": 7, "coverage": 6,             │ 264a3ca1-7bcc-4679-b80eb │
+│                         │                                           │ "coherence": 7}                                                                                  │                          │
+└─────────────────────────┴───────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────┴──────────────────────────┘
+```
+And to download to a JSONL file, use `log10 feedback download --task_id 04405cbc-3420-4901-97b6 --file feedback.jsonl`
+
+To leverage your feedback and use AI to generate more feedback automatically, use [`log10 feedback predict`](#log10-feedback-predict). Please refer to this [doc](https://log10.io/docs/feedback/auto_feedback) for more details. 
+
+## CLI References
+
+### Completions
```

Comment:
Good for now, but we may have to auto gen these in the future: https://stackoverflow.com/questions/57810659/automatically-generate-all-help-documentation-for-click-commands